docs/tutorials/ssp_profile_catalog_authoring/ssp_profile_catalog_authoring.md

```diff
@@ -227,15 +270,33 @@ Add prose here for required Section: Needed Extra
 
 ## Control A Guidance
 
-This is A Guidance.
+Control A prose
+
```

Comment:
The text should be "Control level guidance"

Reply:
fixed